packages/plugin-flex/src/commands/flex/plugins/start.ts

```diff
@@ -92,14 +93,22 @@ export default class FlexPluginsStart extends FlexPlugin {
         await this.checkPlugin(pluginNames[i]);
       }
 
-      // Start flex start once
-      const flexStartScript: StartScript = await this.runScript('start', ['flex', ...flexArgs]);
-
       // Now spawn each plugin as a separate process
+      const pluginConfig: PluginConfig = {};
```

Comment:
Use `Record<string, unknown>` for the time being

Reply:
The problem is if we do this then we can no longer access `pluginConfig[pluginNames[i]].port` as needed on line 111 since the type is `unknown`... Solution is to either make another record that the ports are loaded into as numbers, like with `const portConfig: Record<string, string> = {};`

Or we could make an interface like:
`interface PluginPortConfig {
  port: number;
}`
and make this record on line 97 `const pluginConfig: Record<string, PluginPortConfig> = {}`

Or maybe there's another solution I'm not thinking of?

Reply:
You do `pluginConfig[pluginNames[i]].port as number` on that line

Reply:
`pluginConfig[pluginNames[i]]` doesn't have object `.port` since it is of type `unknown`